### PR TITLE
Fix-EZP-28398: Using Assetic versioning breaks Platform Admin UI

### PR DESCRIPTION
**Pull request by kmadejski**
JIRA issue: https://jira.ez.no/browse/EZP-28398

Todo:
- [x] Rebase on top of master once #927 is merged

### CODE REVIEW

**Comment by webhdx** [comment]
Ugh, let me have a coffee or two before trying to understand regexp magic happening here.

File: ApplicationConfig/Providers/RootInfo.php [hunk 1]
```diff
@@ -48,8 +48,24 @@ public function getConfig()
...
-            'ckeditorPluginPath' => $this->assetsPackages->getUrl($this->externalAssetsDirectory) . '/vendors/',
+            'assetRoot' => $this->getAssetUrl('/'),
+            'ckeditorPluginPath' => $this->getAssetUrl($this->externalAssetsDirectory) . '/vendors/',
         ];
     }
+
+    /**
+     * Returns asset URL without version (if any).
+     *
+     * @param $path
```
Comment (adamwojs) [inline ApplicationConfig/Providers/RootInfo.php]: Nitpick: Missing type hint for `$path`

File: Loader/ComboLoader.php [hunk 2]
```diff
@@ -205,4 +209,23 @@ private function getModuleSetting($module, $setting)
     {
         return $this->configResolver->getParameter("yui.modules.$module.$setting", 'ez_platformui');
     }
+
+    /**
+     * Removes version string (if any) from file paths.
+     *
+     * @param array $files
+     * @param $version
```
Comment (adamwojs) [inline Loader/ComboLoader.php]: Nitpick: Missing type hint for `$version`

File: Loader/Loader.php [hunk 3]
```diff
@@ -21,22 +21,24 @@
...
      *
      * @param array $files File name array
+     * @param string $version Version
      *
      * @return string The content of the combined files
      *
      * @throws \eZ\Publish\Core\Base\Exceptions\NotFoundException
      */
-    public function combineFilesContent(array $files);
+    public function combineFilesContent(array $files, $version);
```
Comment (adamwojs) [inline Loader/Loader.php]: Q: Does  `$version` parameter needs default value from BC point of view?

File: Loader/Loader.php [hunk 4]
```diff
@@ -21,22 +21,24 @@
...
      * @param array $files File name array
+     * @param string $version Version
      *
      * @return string The content type of the combined files
      *
      * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException
      * @throws \eZ\Publish\Core\Base\Exceptions\NotFoundException
      */
-    public function getCombinedFilesContentType(array $files);
+    public function getCombinedFilesContentType(array $files, $version);
```
Comment (adamwojs) [inline Loader/Loader.php]: Q: Does $version parameter needs default value from BC point of view?

**Pull request merged**